core/src/main/java/com/datastax/oss/driver/api/core/loadbalancing/LoadBalancingPolicy.java

```diff
@@ -32,12 +32,20 @@
    * Initializes this policy with the nodes discovered during driver initialization.
    *
    * <p>This method is guaranteed to be called exactly once per instance, and before any other
-   * method in this class.
+   * method in this class. At this point, the driver has successfully connected to one of the
+   * contact points, and performed a first refresh of topology information (by default, the contents
+   * of {@code system.peers}) to find out which other nodes exist in the cluster.
    *
-   * @param nodes the nodes discovered by the driver when it connected to the cluster. When this
-   *     method is invoked, their state is guaranteed to be either {@link NodeState#UP} or {@link
-   *     NodeState#UNKNOWN}. Node states may be updated concurrently while this method executes, but
-   *     if so you will receive a notification.
+   * @param nodes the nodes in the cluster. This method must call {@link
```

Comment:
The parameter description is a bit too long imo, specially considering that it will be rendered in fixed-size font.

Reply:
OK I moved it to the main description.